### PR TITLE
tiltfile: add yaml function

### PR DESCRIPTION
**Pull request by jazzdan**
So that you can take a Starlark string and pass it in to `k8s_yaml`, as demonstrated in the unit test.

This is the last part of #894.

### CODE REVIEW

File: docs/api/api.py [hunk 1]
```diff
@@ -113,3 +113,6 @@ def helm(pathToChartDir: Union[str, LocalPath]) -> Blob:
 
 def fail(msg: str) -> None:
   """Raises an error that cannot be intercepted. Can be used anywhere in a Tiltfile."""
+
+def blob(input: str) -> Blob:
```
Comment (nicks) [inline docs/api/api.py]: i'm still concerned that `blob` is a bad name for this function, but not sure i can come up with a better one

File: internal/tiltfile/files.go [hunk 2]
```diff
@@ -327,3 +327,13 @@ func (s *tiltfileState) helm(thread *starlark.Thread, fn *starlark.Builtin, args
 
 	return newBlob(string(yaml)), nil
 }
+
+func (s *tiltfileState) blob(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var input starlark.String
+	err := starlark.UnpackArgs(fn.Name(), args, kwargs, "path", &input)
```
Comment (nicks) [inline internal/tiltfile/files.go]: "path" -> "input" ?

File: internal/tiltfile/files.go [hunk 3]
```diff
@@ -327,3 +327,13 @@ func (s *tiltfileState) helm(thread *starlark.Thread, fn *starlark.Builtin, args
...
+	if err != nil {
+		return nil, fmt.Errorf("Argument 0 (path): %v", err)
```
Comment (nicks) [inline internal/tiltfile/files.go]: i don't think you need this fmt.Error? UnpackArgs should already give you a far better and more descriptive message.

**Comment by nicks** [comment]
i guess the Go way to do this would be to have k8s_yaml accept a YAML type, and make the conversion from blob -> yaml implicit. So the function would be `yaml('string')`? but if you want to go with `blob`, that's ok too

**Comment by jazzdan** [comment]
@nicks I like that approach better, I'm going to do that.

**Pull request merged**